wopiserver/values.yaml

```
@@ -4,7 +4,7 @@ image:
   repository: cs3org/wopiserver
   tag: v8.3.3
   pullPolicy: IfNotPresent
-
+  pullSecrets: []
```

```suggestion
  pullSecrets: 
    []
```

to harmonize with the rest of the changes in this PR?